spec.html

```diff
@@ -20705,19 +21550,39 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
         1. Perform ! MakeConstructor(_F_, *false*, _proto_).
         1. If |ClassHeritage_opt| is present, set _F_.[[ConstructorKind]] to ~derived~.
         1. Perform ! CreateMethodProperty(_proto_, *"constructor"*, _F_).
-        1. If |ClassBody_opt| is not present, let _methods_ be a new empty List.
-        1. Else, let _methods_ be NonConstructorMethodDefinitions of |ClassBody|.
-        1. For each |ClassElement| _m_ of _methods_, do
-          1. If IsStatic of _m_ is *false*, then
-            1. Let _status_ be PropertyDefinitionEvaluation of _m_ with arguments _proto_ and *false*.
+        1. If |ClassBody_opt| is not present, let _elements_ be a new empty List.
+        1. Else, let _elements_ be NonConstructorMethodDefinitions of |ClassBody|.
+        1. Let _instanceFields_ be a new empty List.
+        1. Let _staticFields_ be a new empty List.
+        1. For each |ClassElement| _e_ of _elements_, do
+          1. If IsStatic of _e_ is *false*, then
+            1. Let _field_ be ClassElementEvaluation of _e_ with arguments _proto_ and *false*.
           1. Else,
-            1. Let _status_ be PropertyDefinitionEvaluation of _m_ with arguments _F_ and *false*.
-          1. If _status_ is an abrupt completion, then
+            1. Let _field_ be ClassElementEvaluation of _e_ with arguments _F_ and *false*.
+          1. If _field_ is an abrupt completion, then
             1. Set the running execution context's LexicalEnvironment to _env_.
-            1. Return Completion(_status_).
+            1. Set the running execution context's PrivateEnvironment to _outerPrivateEnvironment_.
+            1. Return Completion(_field_).
+          1. If _field_ is a ClassFieldDefinition Record, then
+            1. If IsStatic of _e_ is *false*, append _field_ to _instanceFields_.
+            1. Else, append _field_ to _staticFields_.
         1. Set the running execution context's LexicalEnvironment to _env_.
         1. If _classBinding_ is not *undefined*, then
           1. Perform _classScope_.InitializeBinding(_classBinding_, _F_).
+        1. Set _F_.[[Fields]] to _instanceFields_.
+        1. If PrivateBoundIdentifiers of |ClassBody| contains a Private Name _P_ such that _P_.[[Kind]] is either ~method~ or ~accessor~ and _P_.[[Brand]] is _proto_, then
```

Comment:
PrivateBoundIdentifiers returns a list of Strings; it can't possibly contain a Private Name.

Reply:
Obviated by https://github.com/tc39/ecma262/pull/1668/commits/558a63cbec0e6d0de22091664de3f17e0e4eccd7.